### DISCUSSION TITLE
KeeShare empty

### DISCUSSION THREAD

**Discussion by smutel** [keepassxreboot/keepassxc#7946] [Q&A] [answered]
Hello,

I would like to setup KeeShare between two databases.
I did it one time and it worked but I did something wrong, I assume that KeePassXC did'nt like when I tried to remove the shared folder.

So I removed everything and start the KeeShare process from scratch but now the KeeShare folder on the target is empty. The synced icon is green and everything seems to be setup correctly but the folder is still empty.

Thanks.

**Comment by droidmonkey** [comment, marked as answer]
Start from a new database and a new keeshare file. The main database remembers when you delete things and will delete entries from a share that gets reconnected, even if you don't want that to happen. There is an open issue tracking this poor behavior.

**Reply by smutel** [reply]
I created my target database from scratch and it is working again. Thanks.
